ingestion/get_data.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import pandas as pd
+from sodapy import Socrata
+
+# Dictionary of cities and their corresponding API information
+META_CITY = {"new_york":("socrata", "data.cityofnewyork.us", "dm9a-ab7w"),
+            "chicago": ("socrata", "data.cityofchicago.org", "ydr8-5enu"),
+            "mesa": ("socrata", "data.mesaaz.gov", "2gkz-7z4f"),
+            "la": ("socrata", "data.lacity.org", "nbyu-2ha9"),
+            "austin": ("socrata", "data.austintexas.gov", "3syk-w9eu"),
+            "philly_one": ("non_socrata", "https://phl.carto.com/api/v2/sql?filename=permits&format=csv&skipfields=cartodb_id,the_geom,the_geom_webmercator&q=SELECT%20*,%20ST_Y(the_geom)%20AS%20lat,%20ST_X(the_geom)%20AS%20lng%20FROM%20permits%20WHERE%20permitissuedate%20%3E=%20%272016-01-01%27"),
+            "philly_two": ("non_socrata", "https://opendata-downloads.s3.amazonaws.com/opa_properties_public.csv")}
```

Comment:
I think there's an API for philly two: https://cityofphiladelphia.github.io/carto-api-explorer/#opa_properties_public

Can you guys pull from there instead of the csv?

also can philly_one and philly_two be named something more informative